src/MSNMoney.py

```diff
@@ -1,46 +1,38 @@
-import logging
-from lxml import html
-
-def isfloat(value):
-  if value is None:
-    return False
-  try:
-    float(value)
-    return True
-  except ValueError:
-    return False
+import json
 
 class MSNMoney:
-  BASE_URL = 'https://www.msn.com/en-us/money/stockdetails/analysis?symbol={}'
-  PE_HIGH_KEY = 'P/E Ratio 5-Year High'
-  PE_LOW_KEY = 'P/E Ratio 5-Year Low'
-
-  @classmethod
-  def construct_url(cls, ticker_symbol,):
-    url = MSNMoney.BASE_URL.format(ticker_symbol)
-    return url
+  TICKER_URL = 'https://services.bingapis.com/contentservices-finance.csautosuggest/api/v1/Query?query={}&market=en-us'
+  BASE_URL = 'https://services.bingapis.com/contentservices-finance.financedataservice/api/v1/KeyRatios?stockId={}'
```

Comment:
This isn't really a base URL anymore. Can we instead call this KEY_RATIOS_URL or something?

Reply:
Done